### PR TITLE
Add override gas price

### PR DESCRIPTION
**Pull request by pgebheim**

### CODE REVIEW

**Review by robert-e-davidson3 — APPROVED** [review]
lgtm

**Comment by bthaile** [comment]
How is the UI suppose to use this to set gasPrice?

**Comment by pgebheim** [comment]
```
augur.provider.overrideGasPrice = theGasPrice;
```

Can make a helper for that

```
augur.gasPrice = theGasPrice
``` 

:-D

File: packages/augur-core/source/libraries/EthersFastSubmitWallet.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class EthersFastSubmitWallet extends ethers.Wallet {
             delete transaction.from;
             // Add 10% gas to account for improper estimations and limit to block gas limit
             transaction.gasLimit = gasEstimate.add(gasEstimate.div(10));
-            if (transaction.gasLimit.gt(7500000)) transaction.gasLimit = new ethers.utils.BigNumber(7500000);
+            if (transaction.gasLimit.gt(6900000)) transaction.gasLimit = new ethers.utils.BigNumber(6900000);
```
Comment (pgebheim) [inline packages/augur-core/source/libraries/EthersFastSubmitWallet.ts]:
```suggestion
            if (transaction.gasLimit.gt(7500000)) transaction.gasLimit = new ethers.utils.BigNumber(7500000);
```

**Pull request merged**